### DISCUSSION TITLE
Georgian audio samples

### DISCUSSION THREAD

**Discussion by beqabeqa473** [p0p4k/vits2_pytorch#42] [Show and tell]
Hi all.

I've started to train georgian voice to experiment with vits2.
Current progress is 81000 steps, but unfortunately model still has some problems.
Data is 100 minutes long (1000 prompts) though and i understand, that training from scratch might need more training.
What do you think about that?

here is an audio demonstration:
https://tiflo.ge/vits2/

**Comment by p0p4k** [comment]
What is the average length of a single sentence in the training dataset?

**Reply by beqabeqa473** [reply]
The average length of WAV audio files is 5.8 seconds.

**Reply by p0p4k** [reply]
You could try to pick a pretrained English model and start learning from that checkpoint for this task.

**Reply by p0p4k** [reply]
Do you have any other model that you trained on the same data?

**Comment by beqabeqa473** [comment]
yes, i have model trained on vits1, but it is fine-tuned from larger english model.
What i should do to start training from pre-trained english language?
should i add back english symbols or it should be ok to just drop descriminator and start over.

**Reply by p0p4k** [reply]
Load the pretrained vits2 model (you can see some in other discussions and get a checkpoint) in pytorch (state_dict) and then start training the model on your data.
